tests/testthat/test-indentation_linter.R

```diff
@@ -0,0 +1,445 @@
+test_that("indentation linter flags unindented expressions", {
+  linter <- indentation_linter(indent = 2L)
```

Comment:
One more request for extra tests -- currently all the tests just try to match `"Indentation"` in the message, but we should have some tests that ensure the customization is correct